contributors/devel/on-call-build-cop.md

```diff
@@ -31,7 +31,7 @@ longer a part of the mandate.
 ## Workflow
 
 1. Check the Prow batch dashboard: [https://prow.k8s.io/?type=batch](https://prow.k8s.io/?type=batch) 
-to ensure that merges are occurring regularly.
+to ensure that batch jobs are running regularly. It's okay to see occasional flakes.
```

Comment:
Sure, but checking the prow dashboard doesn't tell you that merges are happening.